custom-recipes/dss-plugin-nlp-google-cloud-translation/recipe.json

```diff
@@ -0,0 +1,964 @@
+{
+  "meta": {
+    "label": "Google Cloud Translation",
+    "displayOrderRank": 1,
+    "description": "Translate a text column to another language with  [Google Cloud Translation](https://cloud.google.com/translate/docs/basic/translating-text)",
```

Comment:
May be better to link to the Advanced API doc, if this is what we are using

Reply:
Perhaps need to resync with Adrien & Krishna to validate.

From my end, it's better to use v3 (Advanced) now, since it works out of the box in the same way as v2. It means future additions of v3-specific features such as custom models will be easier.

WDYT?

Reply:
We are using the feature of the v2 so I would link to the v2 ie Basic

Reply:
works for me